app/services/idv/steps/ipp/state_id_step.rb

```diff
@@ -8,19 +8,47 @@ def call
           Idv::StateIdForm::ATTRIBUTES.each do |attr|
             flow_session[:pii_from_user][attr] = flow_params[attr]
           end
+
+          # Accept Date of Birth from both memorable date and input date components
+          fp = flow_params&.[](:dob)
```

Comment:
What does `fp` stand for here in reference to the `:dob` value? I find that abbreviations can be difficult to decipher, so I'd prefer it to be spelled out in full.

Reply:
Variable was misnamed. I corrected the name, refactored some of this into a static method on `MemorableDateComponent`, and added tests to cover this.